importer/tag/src/test/java/org/itsallcode/openfasttrace/importer/tag/TestTagImporter.java

```diff
@@ -272,8 +272,8 @@ void tagWithMultipleRequiredCoverageWithSpaces()
     @Test
     void tagWithRevision()
     {
-        assertItems(
-                tag("impl~~42", "req~example_name~17"), //
+        // Concatenation required to avoid problems during self-trace
+        assertItems("[impl~~42->req~example_name~17" + "]",
```

Comment:
```suggestion
        assertItems("[impl~~42->req~example_name~17" + "]",
```
:+1: I though about this last night. And that is exactly the solution that I had in mind at the end too. Dead simple and does the trick.